frontend/apps/web/components/jobs/NosqlTable/NosqlTable.tsx

```diff
@@ -366,15 +486,27 @@ function AddNewRecord(props: AddNewRecordProps): ReactElement {
 
 interface GetColumnsProps {
   onDelete(row: Row): void;
+  onDuplicate(row: Row): void;
   transformerHandler: TransformerHandler;
   onEdit(row: Row, index: number): void;
-  collections: string[];
-  onDuplicate(row: Row): void;
+  isDuplicateKey: (
+    newValue: string,
+    schema: string,
+    table: string,
+    currValue?: string
```

Comment:
What is the purpose of the `currValue` here and why is it not enough to just ask: is the <schema, table, newKey> a duplication?

Reply:
answered above